app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def update_post_counter
+    users.posts_counter += 1
+  end
+
+  def recent_comments
+    comments.last(5)
+  end
```

Comment:
- [ ] Because a post should show 5 recent_comments you can order them descendant like:
````ruby
def recent_comments
  comments.order('created_at Desc').limit(5)
end
````